docs/reference/dapis/verify-beacon.md

```diff
@@ -22,67 +22,69 @@ within the
 A `beaconId` for each dAPI is derived from the hash of the provider's Airnode
-address and `templateId`
+address and `templateId`.
 
 A `templateId` is a hash of one of the Airnode's `endpointId` and encoded
-parameters
+parameters.
 
 The dAPI team is running Airseekers which are primarily responsible for updating
 the `beaconId` based on the specification of the dAPI. Providers also update the
 `beaconId` at a higher deviation threshold/heartbeat as a fallback.
 
-To verify that the airnode address of a beaconId belongs to the provider listed
-in the market, refer to the
+To verify that the Airnode address of a `beaconId` belongs to the provider
+listed in the market, refer to the
 [Verify Airnode Addresses](/reference/airnode/latest/developers/verify-airnode-addresses.md)
 section.
 
 ## Verifying beaconId
 
-From the API3 [Market<externalLinkImage/>](https://market.api3.org/dapis),
+From the [API3 Market<externalLinkImage/>](https://market.api3.org/dapis),
 navigate to the parameters page of a dAPI. As an example for
-[ETH/USD on gnosis chain<externalLinkImage/>](https://market.api3.org/dapis/gnosis/ETH-USD/parameters)
+[ETH/USD on Gnosis chain<externalLinkImage/>](https://market.api3.org/dapis/gnosis/ETH-USD/parameters)
 the parameters are as follows:
 
 <img src="./assets/images/dapi-parameters.png"/>
 
-The templateId is calculated by taking the hash of the `encodedParameters` and
+The `templateId` is calculated by taking the hash of the `encodedParameters` and
 `endpointId`.
 
-Following is an ethers.js script to derive the templateId (you need
-[airnode-abi](/reference/airnode/latest/packages/airnode-abi.md) installed):
+Following is an `ethers.js` v5 script to derive the `templateId` (you need
+[`@airnode-abi`](reference/airnode/latest/packages/airnode-abi.md) installed):
 
 ```javascript
 import { encode } from '@api3/airnode-abi';
 import { ethers } from 'ethers';
 
 const encodedParameters = encode(decodedParameters);
-const templateId = hre.ethers.utils.solidityKeccak256(
+const templateId = ethers.utils.solidityKeccak256(
   ['bytes32', 'bytes'],
   [endpointId, encodedParameters]
 );
 ```
 
-Once derived the beaconId is the hash of the `templateId` and Airnode address:
+Once derived the `beaconId` is the hash of the Airnode address and `templateId`:
 
 ```javascript
 import { ethers } from 'ethers';
 
-const beaconId = hre.ethers.utils.solidityKeccak256(
-  ['bytes32', 'address'],
-  [templateId, AirnodeAddress]
+const beaconId = ethers.utils.solidityKeccak256(
+  ['address', 'bytes32'],
+  [AirnodeAddress, templateId]
 );
 ```
 
 ## Verifying beaconSetId
 
 Beacon sets are a collection of `beaconIds` that provide the aggregated median
 value of the underlying `beaconIds`. The `beaconSetId` is a hash of all the
-underlying `beaconIds` and points to the datafeed containing the median value.
-Following is an ethers.js script to derive the `beaconSetId`.
+underlying `beaconIds` and points to the Datafeed containing the median value.
```

Comment:
Normally **Datafeed** would be data feed. It is not an important word such as Airnode. It is not a product or institution name for example.

Reply:
got it ,will fix